### DISCUSSION TITLE
How does a new version get released?

### DISCUSSION THREAD

**Discussion by ShadiNM** [O365/python-o365#1107] [General]
I recently submitted a PR that already got merged (thanks @alejcas)
what are the next steps to get the changes released so that when the library is pulled from Pypi it uses the latest changes. I think it should be 2.0.37 now

**Comment by alejcas** [comment]
I have to push the version, I usually wait a bit so other updates made into the version.

**Reply by ShadiNM** [reply]
how long does that typically take?

**Reply by ShadiNM** [reply]
@alejcas are there any other updates coming that you are waiting for? Otherwise I wonder if it could take months to get this new version released?

**Reply by alejcas** [reply]
Done
